babel.config.js

```diff
@@ -14,12 +14,9 @@ module.exports = function BabelConfigJS(api) {
         useESModules: false,
       },
     ],
-    [
-      "@babel/plugin-proposal-class-properties",
-      {
-        loose: true,
-      },
-    ],
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
```

Comment:
these were required after the upgrade I can imagine.

Reply:
Yes, Storybook needed them